src/internal/query.spec.ts

```diff
@@ -65,6 +66,34 @@ describe("Query class", () => {
     });
   });
 
+  describe("On setAction", () => {
```

Comment:
I would add the test for the action with parent resource conditions, i.e. for the requests like:
```
.attach('relatedResource`, filter => ...)
.where(...)
```

Reply:
do you mean testing together with another `query` param like `addSortCondition` or `setFilterCriteria` or are you talking about `ActionQuery` tests?

Reply:
the first one, together with another query param

Reply:
I see no point in testing `setAction` isolated with some other random method, so I created one test with all conditions together.

Reply:
👍